lib/painted_rabbit/serializer.rb

```diff
@@ -10,7 +10,7 @@ def self.bleh
     @_blah
```

Comment:
I might have missed something, but why the `bleh` and `@_blah`?
Placeholders?

Reply:
Yes - that code wasn't introduced in this PR, it is original to the repo. We plan on refactoring/fixing this in the very next PR.